src/card/card.jsx

```diff
@@ -13,16 +13,42 @@ const Card = React.createClass({
   },
 
   propTypes: {
+    /**
+    * Whether a click on this card component expands the card. Can be set on any child of the Card component.
+    */
     actAsExpander: React.PropTypes.bool,
```

Comment:
also, add default prop for all the bool types, it makes it more expressive in the docs. the code generator can pick them up :D